Packs/HelloWorld/Integrations/HelloWorld/HelloWorld_description.md

```diff
@@ -1,3 +1,3 @@
 ## Hello World
-- This text is markdown
-- Here you should explain how to configure the instance in Demisto
\ No newline at end of file
+- This section explains how to configure the instance of HelloWorld in Cortex XSOAR.
+- To obtain the HelloWorld API Key, ask to the Cortex XSOAR team on the `#dmst-integrations-help` channel on the [Slack DFIR community](https://dfircommunity.slack.com).
```

Comment:
Can we expose this key to everyone? Because HelloWorld is a default integration template in Demisto.

Reply:
Yes, it shouldn't be a problem. The API itself also is on github and they can deploy their own on AWS if they want.

Reply:
Do you want to put it in the description file?